### PR TITLE
[NonPoolSuperfluid] docs

### PR DESCRIPTION
**Pull request by nicolaslara**
## What is the purpose of the change
Adds some docs for the non-pool superfluid feature

## Testing and Verifying
ND

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`?

Where is the change documented?
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [ ] N/A

### CODE REVIEW

File: x/superfluid/README.md [hunk 1]
```diff
@@ -695,9 +716,12 @@ superfluid staking than just LP shares. Each AssetType has a different
 algorithm used to get its "Osmo equivalent value".
 
 We represent different types of superfluid assets as different enums.
-Currently, only enum `1` is actually used. Enum value `0` is reserved
-for the Native staking token for if we deprecate the legacy staking
-workflow to have native staking also go through the superfluid module.
+Currently, enum `1` represent s Gamm shares and enum `2` Concentrated
```
Comment (czarcas7ic) [inline x/superfluid/README.md:719]: "represents s Gamm shares" I assume is a typo

File: x/superfluid/README.md [hunk 2]
```diff
@@ -1048,3 +1072,72 @@ the `IntermediaryAccount` will be slashed by less than the
 TODO - expand on this Uses `lockup` accumulator to find total amount of
 synthetic locks for a given `IntermediaryAccount` (Superfluid Asset +
 Validator pair)
+
+## Pool tokens vs non-pool Tokens
```
Comment (czarcas7ic) [inline x/superfluid/README.md:1076]:
```suggestion
## Pool tokens vs non-pool tokens
```

File: x/superfluid/README.md [hunk 3]
```diff
@@ -1048,3 +1072,72 @@ the `IntermediaryAccount` will be slashed by less than the
...
+
+### Querying delegations
+
+To check the superfluid delegations for a user the following query can be used:
+
+```go
+types.SuperfluidDelegationsByDelegatorRequest{DelegatorAddress: userAddr.String()}
+```
+
+### Rewards
```
Comment (czarcas7ic) [inline x/superfluid/README.md:1126]: Is there anything about limits for how much of osmosis stake can be comprised of the native denom? (or rather, the max percent of rewards that can go to the native token). I think this isn't implemented yet though, just came to mind as I was reading this.

**Review by czarcas7ic — APPROVED** [review]

File: x/superfluid/README.md (hunk 1 above)
Comment (czarcas7ic) [reply x/superfluid/README.md]: https://github.com/osmosis-labs/osmosis/pull/8331/commits/7a62c33feac4dfcb8fbe33d95634d71e70d6edf0

File: x/superfluid/README.md (hunk 3 above)
Comment (czarcas7ic) [reply x/superfluid/README.md:1126]: Left a comment in https://github.com/osmosis-labs/osmosis/pull/8335

We can just add to the docs with that PR

**Review by PaddyMc — APPROVED** [review]

**Pull request merged**